_includes/head.html

```diff
@@ -6,7 +6,7 @@
   <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
   <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
 
-  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: site.baseurl }}">
+  <link rel="stylesheet" href="{{ "/assets/stylesheets/main.css" | prepend: site.baseurl }}">
```

Comment:
> We don't have an images folder ;)

Yet. Oh, just look at this PR :smirk:

`/assets` is one folder.

`/assets/stylesheets` and `/assets/images` is still only one folder, if you cound from root, however `/css` and `/images` is two folders counting from the same root.

I can however see that you are correct and that Jekyll does this too themselves (https://github.com/jekyll/jekyll/tree/master/site)